.gitlab-ci.yml

```
@@ -25,12 +25,12 @@ production:
     stage: production
     image: ruby:latest
     script:
-        - echo --env-file "ACCOUNT_SID=${ACCOUNT_SID} \n" >> .env
-        - echo --env-file "AUTH_TOKEN =${AUTH_TOKEN} \n" >> .env
-        - echo --env-file "API_KEY=${API_KEY} \n" >> .env
-        - echo --env-file "API_SECRET=${API_SECRET} \n" >> .env
-        - echo --env-file "PHONE_NUMBER =${PHONE_NUMBER} \n" >> .env
-        - echo --env-file "sengrid_api =${sengrid_api} \n" >> .env
+        - echo "ACCOUNT_SID=$ACCOUNT_SID" >> .env
+        - echo "AUTH_TOKEN=$AUTH_TOKEN" >> .env
+        - echo "API_KEY=$API_KEY" >> .env
+        - echo "API_SECRET=$API_SECRET" >> .env
+        - echo "PHONE_NUMBER=$PHONE_NUMBER" >> .env
+        - echo "sengrid_api=$sengrid_api" >> .env
```

Curious if this should be:
```suggestion
        - echo "SENDGRID_API=$SENDGRID_API" >> .env
```
with the `d` and capitalized.